build.assets/Dockerfile-centos7

```diff
@@ -53,6 +53,7 @@ RUN yum install --nogpgcheck -y \
 
 # Needed for boring-rs
 ENV CMAKE=cmake3
+RUN ln -s /usr/bin/cmake3 /usr/local/bin/cmake
```

Comment:
It should work, but why is the symlink not in `/usr/bin/` where the binary is?

Reply:
`/usr/bin` is owned by the package manager. If we put the symlink in `/usr/bin`, it is possible that old cmake (is that version 2?) could be installed after, but it would follow that symlink and overwrite cmake3. With `/usr/local/bin` in the path before `/usr/bin`, this means even if some package installs `/usr/bin/cmake`, we'll still get what we expect - cmake3